Update note to provide Pull Provider feature (#1721)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc

NOTE: The Katello agent is deprecated and will be removed in a future {Project} version.
Migrate your workloads to use the remote execution feature to update clients remotely.
For more information, see {ManagingHostsDocURL}Migrating_From_Katello_Agent_to_Remote_Execution_managing-hosts[Migrating from Katello Agent to Remote Execution] in _{ManagingHostsDocTitle}_.

> With the release of {Project} {ProjectVersion}, we are providing support for a pull-based provider, an agent-based tool as a replacement for the Katello agent.

Reviewer: ```suggestion
With the release of {Project} {ProjectVersion}, we provide support for a pull-based client as a replacement for the Katello agent.
```

Reviewer 2: I would drop {ProjectVersion} altogether. It simplifies and shortens the sentence a lot.
```suggestion
{Project} now supports a pull-based provider as a replacement for the Katello agent.
```

> With the release of {Project} {ProjectVersion}, we are providing support for a pull-based provider, an agent-based tool as a replacement for the Katello agent.

Reviewer: It is and this is supposed to be an additional mention in Upgrading.
